### ISSUE TITLE
Docs, NodeJS version need update

### ISSUE THREAD

**Issue by mmmcorpsvit** [mirumee/saleor#2889]
Just fix https://saleor.readthedocs.io/en/latest/gettingstarted/installation-windows.html
Node from 8 to 10+ version

**Comment by NyanKiyoshi** [comment]
Looks like the docs are a few weeks out of sync.

**Comment by maarcingebala** [comment]
I've just noticed that our docs haven't been released since a month due to a broken build on `readthedocs`. I'll take a look what might be the problem. Thanks for pointing that out.

**Comment by maarcingebala** [comment]
Docs are outdated, because ReadTheDocs is unable to install `pip install yarl==1.2.6`:
https://readthedocs.org/projects/saleor/builds/7801602/

I have no idea why it happens, but it seems that this is [a known problem on RTD](https://github.com/rtfd/readthedocs.org/issues/4246).

Another problem is that we're building whole virtualenv from Saleor's `requirements.txt` on RTD which is not necessary to build the documentation. I don't have access to RTD admin panel to change anything (only @patrys has access from what I see). Workaround recommended in the issue I linked above is to add a new, empty requirements file and point to it in RTD admin.

To fix the current issue, I can try removing the version number of `yarl` in our `requirements.txt`. This should make `pip` on RTD install any version. I tested it with my own fork and it worked.
